### PR TITLE
[mini-PR] update path to example batch script on Summit

### PR DESCRIPTION
**Pull request by MaxThevenet**

### CODE REVIEW

File: Docs/source/running_cpp/platforms.rst [hunk 1]
```diff
@@ -66,4 +66,5 @@ regime), the following set of parameters provided good performance:
 * **Two `128x128x128` grids per GPU**, or **one `128x128x256` grid per GPU**.
 
 A batch script with more options regarding profiling on Summit can be found at
-:download:`Summit batch script<../../../Examples/Tests/gpu_test/script_profiling.sh>`
\ No newline at end of file
+:download:`Summit batch script<../../../Examples/Tools/script_profiling_summit.sh>`
```
Comment (RemiLehe) [inline Docs/source/running_cpp/platforms.rst]: Did you mean `../../../Tools/batchScripts/script_profiling_summit.sh`?

**Comment by MaxThevenet** [comment]
Oops, sorry, should be fixed now.

File: Docs/source/running_cpp/platforms.rst [hunk 2]
```diff
@@ -66,4 +66,6 @@ regime), the following set of parameters provided good performance:
 * **Two `128x128x128` grids per GPU**, or **one `128x128x256` grid per GPU**.
 
 A batch script with more options regarding profiling on Summit can be found at
-:download:`Summit batch script<../../../Examples/Tests/gpu_test/script_profiling.sh>`
\ No newline at end of file
+:download:`Summit batch script<../../../Examples/Tools/batchScripts/script_profiling_summit.sh>`
```
Comment (RemiLehe) [inline Docs/source/running_cpp/platforms.rst]: I think that `Examples/Tools` should be replaced by simply `Tools`, correct?

**Comment by MaxThevenet** [comment]
I am so rusted after 2 weeks of inactivity :/
3 commits for a single line changed is a bit excessive, so I will close this PR, and open a new one instead at https://github.com/ECP-WarpX/WarpX/pull/552.

**Pull request closed without merging**